src/libraries/Microsoft.Win32.Registry.AccessControl/src/Microsoft.Win32.Registry.AccessControl.csproj

```diff
@@ -1,17 +1,26 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
-    <TargetFrameworks>netstandard2.0-windows;netstandard2.0;net461-windows</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent);netstandard2.0-windows;netstandard2.0;net461-windows</TargetFrameworks>
```

Comment:
Just wondering, why does this one need both `netstandard2.0-windows;netstandard2.0`, isn't the latter strictly a superset of the former?

Reply:
Actually it's the other way around. The netstandard2.0 (and with this change the NetCoreAppCurrent) asset just throws `PlatformNotSupportedExceptions` and the netstandard2.0-windows asset (and the NetCoreAppCurrent-windows) contains the actual code.

Note that the reference assembly is the same for both TFMs - the API set doesn't change based on the OS the source assembly is targeting.